.github/workflows/release.yml

```diff
@@ -93,9 +93,19 @@ jobs:
           EDITOR: vi
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+  docs:
+    needs: [release, version]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: ./.github/ci/publish_docs.sh
+      env:
+        TIGERBEETLE_COMMIT: ${{needs.version.outputs.version}}
+        TIGERBEETLE_DOCS_DEPLOY_KEY: ${{ secrets.TIGERBEETLE_DOCS_DEPLOY_KEY }}
```

Comment:
You'll need to add `TIGERBEETLE_DOCS_DEPLOY_KEY` to the `secrets:` section under `workflow_call` and pass it in in `pipeline-main.yml` too